libs/x-content/src/main/java/org/opensearch/common/xcontent/cbor/CborXContent.java

```diff
@@ -75,6 +77,8 @@ public static XContentBuilder contentBuilder() throws IOException {
         cborFactory.configure(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT, false);
         cborFactory.configure(JsonParser.Feature.STRICT_DUPLICATE_DETECTION, true);
         cborFactory.setStreamReadConstraints(StreamReadConstraints.builder().maxStringLength(DEFAULT_MAX_STRING_LEN).build());
+        cborFactory.configure(StreamReadFeature.USE_FAST_DOUBLE_PARSER.mappedFeature(), true);
```

Comment:
Please extract the property as the constant (otherwise it is lost in code). Also, please name it `opensearch.xcontent.use_fast_double_writer` or `opensearch.xcontent.fast_double_writer.enabled` (so becomes clear it is related to XContent).

```
public static final int USE_FAST_DOUBLE_WRITER = Boolean.getBoolean("opensearch.xcontent.use_fast_double_writer");

```

Reply:
Makes sense. I've added the property in a constant as suggested. Followed the same convention as used for the `opensearch.xcontent.string.length.max` system property added earlier.